### ISSUE TITLE
Inf loss

### ISSUE THREAD

**Issue by moodhiaj** [cestwc/pointer-generator-model-pytorch#1]
Hello, I'm trying to use what you did here but I keep getting inf validation loss, any ideas why?
